core/templates/components/common-layout-directives/common-elements/promo-bar.directive.html

```diff
@@ -1,6 +1,6 @@
-<div class="oppia-toast-container toast-top-center" ng-if="$ctrl.promoBarIsEnabled && $ctrl.promoIsVisible">
+<div class="oppia-toast-container toast-top-center" *ngIf="promoBarIsEnabled && promoIsVisible">
```

Comment:
Do we need to change the name of this file as well? (Not sure!)

Reply:
I think it is not needed.

Reply:
@suryasiriki4, I had a discussion on this with @suryasiriki4 and I confirm that we expect to have `component` in the template filename.